lib/api_guard/jwt_auth/authentication.rb

```diff
@@ -69,15 +93,16 @@ def define_current_resource_accessors(resource)
       #
       # Also, set "current_{{resource_name}}" method and "@current_{{resource_name}}" instance variable
       # for accessing the authenticated resource
-      def authenticate_token
+      def authenticate_token(multiple = false)
         return unless decode_token
 
         resource = find_resource_from_token(@resource_name.classify.constantize)
-
         if resource && valid_issued_at?(resource) && !blacklisted?(resource)
-          define_current_resource_accessors(resource)
+          @current_resource = define_current_resource_accessors(resource)
         else
-          render_error(401, message: I18n.t('api_guard.access_token.invalid'))
+          if multiple === false
+            render_error(401, message: I18n.t('api_guard.access_token.invalid'))
+          end
```

Comment:
We can remove this `else` logic completely. I may have forgot to remove this logic. We already have logic at the end of the `authenticate_and_set_resource` method to render 401 error when resource not found.